cmd/check_container.go

```diff
@@ -236,9 +251,12 @@ func init() {
 	checkContainerCmd.Flags().String("pyxis-api-token", "", "API token for Pyxis authentication")
 	viper.BindPFlag("pyxis_api_token", checkContainerCmd.Flags().Lookup("pyxis-api-token"))
 
-	checkContainerCmd.Flags().String("pyxis-host", certification.DefaultPyxisHost, "Host to use for Pyxis submissions.")
+	checkContainerCmd.Flags().String("pyxis-host", "", fmt.Sprintf("Host to use for Pyxis submissions.\nThis will override Pyxis Env."))
```

Comment:
- Should we add an example format for this value here?
- Should we extend the message to discourage use of this flag for most use cases?

Reply:
I almost want to just say: "Only set this if you know what you're doing"